### DISCUSSION TITLE
NextJs+ExpressJS/TypeScript

### DISCUSSION THREAD

**Discussion by jacksmith1984** [vercel/next.js#37843] [Help]
Hello Team,
We have Vercel+Next.js application running well 
Now we want to install Bablic third party app using following code.
https://help.bablic.com/en/articles/616974-seo-for-nodejs

If we need to install the above reference code , Can you please suggest the best way to implement this?
Since it is a express js or typescript , will it work under the same next.js+vercel or do we need to use custom server?
How can we make the custom server run the app that can be used inside our Next.js+vercel code ?

Thank You

**Comment by jacksmith1984** [comment]
Any help woukd be appreciated .

Thank You

**Comment by GabenGar** [comment]
Use [next-seo](https://github.com/garmeeh/next-seo) instead.

**Comment by jacksmith1984** [comment]
Hi,
We are actually using it for language translation and not mainly for SEO.

Thank you

**Reply by GabenGar** [reply]
It doesn't do anything for language translation by itself though.
